src/python/pants/init/extension_loader.py

```diff
@@ -144,4 +144,8 @@ def invoke_entrypoint(name):
   if subsystems:
     build_configuration.register_subsystems(subsystems)
 
+  rules = invoke_entrypoint('rules')
+  if rules:
+    build_configuration.register_rules(rules)
```

Comment:
I'd like to see some unit tests for this in `test_extension_loader.py`

Reply:
It's only 3 lines long... seems like overkill?

Reply:
Fair. My thought was that as this is a plugin-writer facing change, it should have tests. If we're currently thinking of it as being experimental, then it's fine without. It'd be nice if that were noted somehow though.

Reply:
It's definitely covered by tests, in that the rest of this change relies on it working... it just doesn't seem worth a unit test. It's definitely not code with high cyclomatic complexity, for example.

Reply:
Thanks!